scripts/hail_batch/main.py

```diff
@@ -0,0 +1,28 @@
+"""Run HGDP_1KG_PCALoadingsGenerate.py using the analysis runner."""
+
+import os
+import hail as hl
+import hailtop.batch as hb
+from analysis_runner import dataproc
+
+OUTPUT = os.getenv('OUTPUT')
+assert OUTPUT
+
+hl.init(default_reference='GRCh38')
+
+service_backend = hb.ServiceBackend(
+    billing_project=os.getenv('HAIL_BILLING_PROJECT'), bucket=os.getenv('HAIL_BUCKET')
+)
+
+batch = hb.Batch(name='generate PCA loadings', backend=service_backend)
+
+dataproc.hail_dataproc_job(
+    batch,
+    f'HGDP_1KG_PCALoadingsGenerate.py --output={OUTPUT}',
+    max_age='10h',
+    num_secondary_workers='100'
+    packages=['click', 'gnomad'],
```

Comment:
Do you actually need `gnomad`?

Reply:
No, that's a good point. I think that was a holdover from a previous script (I used it in my hail dataproc command).